sky/serve/service.py

```diff
@@ -145,7 +140,7 @@ def _start(service_name: str, tmp_task_yaml: str, job_id: int):
         controller_job_id=job_id,
         policy=service_spec.policy_str(),
         auto_restart=service_spec.auto_restart,
-        requested_resources=requested_resources,
```

Comment:
Why can't we still save the `requested_resources`, which can be a list or set of resources? It might be easier to process the resources information to have a object than a string?

Reply:
After an offline discussion, since we still need to store the number of nodes, we decided to keep the current implementation.